ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ApprovedDeferrals.tsx

```diff
@@ -83,40 +87,30 @@ function ApprovedDeferrals() {
             ),
         [searchFilter, sortOption, page, perPage]
     );
-    const { data, loading, error } = useRestQuery(vulnerabilityExceptionsFn);
+    const { data, loading: isLoading, error } = useRestQuery(vulnerabilityExceptionsFn);
 
+    const tableUIState = getTableUIState({
+        isLoading,
+        isPolling: false,
```

Comment:
Since polling is an edge case throughout the app, can we make this property optional, defaulting to `false`, so that we don't need to specify it at every call site?

Reply:
Makes sense. Will do